### PR TITLE
Update redirected URLs (part 42)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Extending #14055

## Summary

Updates redirected URLs.

### CODE REVIEW

File: files/en-us/learn/common_questions/available_text_editors/index.md [hunk 1]
```diff
@@ -485,7 +485,7 @@ When you install a new text editor, your OS will probably continue to open text
 - Windows
 
   - [Windows 8](https://support.microsoft.com/en-us/windows?ui=en-US&rs=en-US&ad=US)
-  - [Windows 7](https://windows.microsoft.com/en-us/windows/change-default-programs#1TC=windows-7)
+  - [Windows 7](https://support.microsoft.com/en-gb/windows/change-which-programs-windows-7-uses-by-default-62fd162f-8c82-0436-806f-c60d69dcf495)
```
Comment (OnkarRuikar) [inline files/en-us/learn/common_questions/available_text_editors/index.md:488]:
```suggestion
  - [Windows 7](https://support.microsoft.com/en-us/windows/change-which-programs-windows-7-uses-by-default-62fd162f-8c82-0436-806f-c60d69dcf495)
```

File: files/en-us/learn/common_questions/how_much_does_it_cost/index.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ You can use any of them, as they will have similar functionality, though some ar
...
+For audio files, you can find free software ([Audacity](https://www.audacityteam.org/), [Wavosaur](https://www.wavosaur.com/)), or paying up to a few hundred dollars ([Sony Sound Forge](https://www.magix.com/?utm_source=sonycreativesoftware&utm_medium=referral&utm_campaign=redirect&lang=en&prdt=soundforgepro), [Adobe Audition](https://www.adobe.com/products/audition.html)). Likewise, video-editing software can be free ([PiTiVi](https://www.pitivi.org/), [OpenShot](https://www.openshot.org/) for Linux, [iMovie](https://www.apple.com/imovie/) for Mac), less than $100 ([Adobe Premiere Elements](https://www.adobe.com/us/products/premiere-elements.html)), or several hundred dollars ([Adobe Premiere Pro](https://www.adobe.com/products/premiere.html), [Avid Media Composer](https://www.avid.com/media-composer), [Final Cut Pro](https://www.apple.com/final-cut-pro/)). The software you received with your digital camera may cover all your needs.
```
Comment (OnkarRuikar) [inline files/en-us/learn/common_questions/how_much_does_it_cost/index.md:69]:
```suggestion
For audio files, you can find free software ([Audacity](https://www.audacityteam.org/), [Wavosaur](https://www.wavosaur.com/)), or paying up to a few hundred dollars ([SOUND FORGE](https://www.magix.com/us/music-editing/sound-forge/), [Adobe Audition](https://www.adobe.com/products/audition.html)). Likewise, video-editing software can be free ([PiTiVi](https://www.pitivi.org/), [OpenShot](https://www.openshot.org/) for Linux, [iMovie](https://www.apple.com/imovie/) for Mac), less than $100 ([Adobe Premiere Elements](https://www.adobe.com/us/products/premiere-elements.html)), or several hundred dollars ([Adobe Premiere Pro](https://www.adobe.com/products/premiere.html), [Avid Media Composer](https://www.avid.com/media-composer), [Final Cut Pro](https://www.apple.com/final-cut-pro/)). The software you received with your digital camera may cover all your needs.
```

File: files/en-us/learn/common_questions/how_much_does_it_cost/index.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ You can use any of them, as they will have similar functionality, though some ar
...
+For audio files, you can find free software ([Audacity](https://www.audacityteam.org/), [Wavosaur](https://www.wavosaur.com/)), or paying up to a few hundred dollars ([SOUND FORGE](https://www.magix.com/us/music-editing/sound-forge/), [Adobe Audition](https://www.adobe.com/products/audition.html)). Likewise, video-editing software can be free ([PiTiVi](https://www.pitivi.org/), [OpenShot](https://www.openshot.org/) for Linux, [iMovie](https://www.apple.com/imovie/) for Mac), less than $100 ([Adobe Premiere Elements](https://www.adobe.com/us/products/premiere-elements.html)), or several hundred dollars ([Adobe Premiere Pro](https://www.adobe.com/products/premiere.html), [Avid Media Composer](https://www.avid.com/media-composer), [Final Cut Pro](https://www.apple.com/final-cut-pro/)). The software you received with your digital camera may cover all your needs.
```
Comment (teoli2003) [inline files/en-us/learn/common_questions/how_much_does_it_cost/index.md:69]: The capitalization of Sound Forge looks more like a marketing trick on their web page.
```suggestion
For audio files, you can find free software ([Audacity](https://www.audacityteam.org/), [Wavosaur](https://www.wavosaur.com/)), or paying up to a few hundred dollars ([Sound Forge](https://www.magix.com/us/music-editing/sound-forge/), [Adobe Audition](https://www.adobe.com/products/audition.html)). Likewise, video-editing software can be free ([PiTiVi](https://www.pitivi.org/), [OpenShot](https://www.openshot.org/) for Linux, [iMovie](https://www.apple.com/imovie/) for Mac), less than $100 ([Adobe Premiere Elements](https://www.adobe.com/us/products/premiere-elements.html)), or several hundred dollars ([Adobe Premiere Pro](https://www.adobe.com/products/premiere.html), [Avid Media Composer](https://www.avid.com/media-composer), [Final Cut Pro](https://www.apple.com/final-cut-pro/)). The software you received with your digital camera may cover all your needs.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Thanks, only one nit and we can merge this.

File: files/en-us/learn/common_questions/how_much_does_it_cost/index.md (hunk 3 above)
Comment (OnkarRuikar) [reply files/en-us/learn/common_questions/how_much_does_it_cost/index.md:69]: On their website they are using all caps for the product name: https://www.magix.com/in/music/sound-forge/sound-forge-pro/

Comment (OnkarRuikar) [inline files/en-us/learn/common_questions/how_much_does_it_cost/index.md:69]:
```suggestion
For audio files, you can find free software ([Audacity](https://www.audacityteam.org/), [Wavosaur](https://www.wavosaur.com/)), or paying up to a few hundred dollars ([Sound Forge](https://www.magix.com/us/music-editing/sound-forge/), [Adobe Audition](https://www.adobe.com/products/audition.html)). Likewise, video-editing software can be free ([PiTiVi](https://www.pitivi.org/), [OpenShot](https://www.openshot.org/) for Linux, [iMovie](https://www.apple.com/imovie/) for Mac), less than $100 ([Adobe Premiere Elements](https://www.adobe.com/us/products/premiere-elements.html)), or several hundred dollars ([Adobe Premiere Pro](https://www.adobe.com/products/premiere.html), [Avid Media Composer](https://www.avid.com/media-composer), [Final Cut Pro](https://www.apple.com/final-cut-pro/)). The software you received with your digital camera may cover all your needs.
```

Comment (teoli2003) [reply files/en-us/learn/common_questions/how_much_does_it_cost/index.md]: Yes, but we don't need to apply their marketing trick here.

**Review by teoli2003 — APPROVED** [review]

**Comment by OnkarRuikar** [comment]
I missed your comment above, now I've changed it to 'Sound Forge'.

**Pull request merged**